jobs/sql/extractions/backup_restore_dates.sql

```diff
@@ -0,0 +1,15 @@
+drop temporary table if exists temp_backup_restore_dates;
+create temporary table temp_backup_restore_dates
+(datetime_type varchar(255),
+datetime       text);
+
+insert into temp_backup_restore_dates
+select 'backup',global_property_value('percona_backup_date', null);
+
+insert into temp_backup_restore_dates
+select 'restore',global_property_value('percona_restore_date', null);
+
+select
+datetime_type,
+datetime
```

Comment:
You could probably make this an actual datetime by casting it as this point, if that is better